### PR TITLE
Square salamander: use mask-image component on home page

### PR DESCRIPTION
**Pull request by modernserf**
## Links
* Remix link https://square-salamander.glitch.me
* Manuscript link https://glitch.manuscript.com/f/cases/3328364/

## GIF/Screenshots:
Screen Shot 2019-04-23 at 2 17 19 PM
Screen Shot 2019-04-23 at 2 17 14 PM
Screen Shot 2019-04-23 at 2 18 25 PM

## Changes:
* use MaskImage component on home page
* tweaks for MaskImage component (add mask5 class, allow hyphenated class names, dont fit to height)
* remove old mask CSS

## How To Test:
* visit https://square-salamander.glitch.me
* verify the masks look correct on featured embed and culture blog links

### CODE REVIEW

File: src/components/images/mask-image.js [hunk 1]
```diff
@@ -4,17 +4,18 @@ import classnames from 'classnames';
...
-const maskClasses = ['mask1', 'mask2', 'mask3', 'mask4'];
+const maskClassesWithDash = ['mask-1', 'mask-2', 'mask-3', 'mask-4', 'mask-5'];
+const maskClasses = maskClassesWithDash.map((className) => className.replace('-', ''));
 
 const MaskImage = ({ maskClass: controlledMaskClass, ...props }) => {
   const randomMaskClass = useRef(sample(maskClasses));
   const maskClass = controlledMaskClass || randomMaskClass.current;
 
-  return <img {...props} alt="" className={classnames(styles.mask, styles[maskClass])} />;
+  return <img alt="" {...props} className={classnames(styles.mask, styles[maskClass.replace('-', '')])} />;
```
Comment (keithk) [inline src/components/images/mask-image.js]: Should we use the image component here?

**Comment by keithk** [comment]
Looks right on the page 🚢

File: src/components/images/mask-image.js (hunk 1 above)
Comment (modernserf) [reply src/components/images/mask-image.js]: Would that serve any particular purpose? it doesn't use any of the features of the <Image> component that aren't inherent to an  tag

Comment (keithk) [reply src/components/images/mask-image.js]: 🤷‍♂️ nope!

**Pull request merged**